### ISSUE TITLE
For dropdown/select need to add aria-label Attribute in schema level to meet WCAG

### ISSUE THREAD

**Issue by vishnu947** [json-editor/json-editor#1457]
### General information

I want to make sure my project meets the WCAG (Web Content Accessibility Guidelines) , But i am getting the Accessibility Alert like this for the **dropdown  [Select has no accessible name, associated <label>, or [title].**

- json-editor version:

"@json-editor/json-editor": "^2.9.1",

### Steps to reproduce the behavior

You can check by using any of the WCAG web testing tools like ANDI, Axe-core Dev Tools in the json-json editor interactive demo  example (https://json-editor.github.io/json-editor/)
![Select dropdown label issue](https://github.com/json-editor/json-editor/assets/99916723/34d7bbfb-5b42-4c77-8b39-0a68cbc8f50c)

As you see in the above screenshot i am seeing for input type="text" there is aria-label attribute associated for waiter but for select dropdown type i am not seeing that.

Direct link to example: https://json-editor.github.io/json-editor/

```javascript
{
  "title": "json schema example",
  "type": "object",
  "properties": {
    "example": {
      "type": "string",
      "description": "This is an example schema.",
      "default": "Please edit me."
    }
  }
}
```

**Comment by germanbisurgi** [comment]
I think that it would be nice and useful to go one step further and improve the general accessibility of the json editor by adding accessibility features when missing and improve/correct existing ones.

Starting with meaningful aria-labels

how it is now:
<img width="712" alt="meaningless-aria-labels" src="https://github.com/json-editor/json-editor/assets/13135260/dc0e9471-6ee8-46bd-bee3-aacd71be9994">

How can it be (more meaningful)
<img width="623" alt="meaningful-aria-labels" src="https://github.com/json-editor/json-editor/assets/13135260/dae8ca83-7606-4e10-8955-d068d46ea357">

@schmunk42 what do you think?

**Comment by schmunk42** [comment]
@germanbisurgi is this addressed by https://github.com/json-editor/json-editor/pull/1465?

**Comment by germanbisurgi** [comment]
Yes

**Comment by germanbisurgi** [comment]
Implemented in #1465

**Comment by vishnu947** [comment]
I am still seeing the Accessibility error for select dropdown:

I am using the latest json-editor/json-editor version :"@json-editor/json-editor": "^2.13.2"

This is the schema and data i am using :
**Schema JSON** :
{
  "title": "Sales Forecasting",
  "type": "object",
  "properties": {
    "cardMapping": {
      "type": "array",
      "format": "table",
      "title": "Card Variables Mapping",
      "items": {
        "type": "object",
        "properties": {
          "value": {
            "type": "string",
            "title": "Variable Name",
            "enum": [
              "ScenarioId",
              "Name",
              "Description",
              "Status",
              "RUN_DATE",
              "SCENARIO_STATUS",
              "SALES_FORECAST",
              "BRAND_ID",
              "SEC_ATTR1",
              "SEC_ATTR2"
            ]
          },
          "label": {
            "type": "string",
            "title": "Display Name"
          }
        }
      }
    }
  },
  "options": {
    "disable_edit_json": "true"
  }
}

**Data JSON** :
{
  "cardMapping": [
    {
      "value": "Name",
      "label": "Scenario Name"
    },
    {
      "value": "ScenarioId",
      "label": "Scenario Id"
    },
    {
      "value": "Status",
      "label": "Status"
    },
    {
      "value": "RUN_DATE",
      "label": "Run Date"
    },
    {
      "value": "SCENARIO_STATUS",
      "label": "Scenario Status"
    },
    {
      "value": "SALES_FORECAST",
      "label": "Forecast Number"
    }
  ]
}


Attaching the screenshot from interactive demo for your reference :
![Accessibilty Issue](https://github.com/json-editor/json-editor/assets/99916723/53c5d772-4659-4229-9296-64776144254d)

**Comment by germanbisurgi** [comment]
@vishnu947 Starting from version 2.14.0, you can use accessibility features. In the screenshot below, you'll see a select input without a label because of the table format, but it has an aria-label attribute to fix this.

<img width="1186" alt="select-aria-label" src="https://github.com/json-editor/json-editor/assets/13135260/bbecfada-147d-4217-821b-ca1dc1597e3e">

<img width="588" alt="scan" src="https://github.com/json-editor/json-editor/assets/13135260/490f3093-ed39-48e6-897a-1e65626e5501">
